### PR TITLE
Aligned section 9 with improved section 6 (for issue #216)

### PR DESCRIPTION
**Pull request by takuki**
Aligned section 9 with improved section 6.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2838,64 +2838,66 @@ <h3>Existing Device Described with a Thing
...
+               interface. In the <a>WoT Scripting API</a>, <a>Consumed Things</a> 
+               are represented by objects.
+               The behavior implementation (that is, the application logic) running in 
+               a <a>Thing</a> can engage in interactions with <a>Consumers</a> by using 
+               the programming language interface provided by the <a>Exposed Thing</a>.
+            </p>
+            <p> A <a>Thing</a> may not necessarily represent a physical device. 
+                <a>Things</a> can also represent a collection of devices, or virtual services 
+                running in a gateway or in the cloud.
+                Likewise, a <a>Consumer</a> may or may not represent
```
Comment (mlagally) [inline index.html]: What does it represent, if it does not represent an application service?

File: index.html [hunk 2]
```diff
@@ -2838,64 +2838,66 @@ <h3>Existing Device Described with a Thing
...
+                Likewise, a <a>Consumer</a> may or may not represent 
+                an application service running on a gateway or cloud.
+                <a>Consumers</a> can also be implemented on edge devices.
+                In <a>Intermediaries</a>, a single <a>Servient</a>
+                simultaneously performs both the roles of a <a>Thing</a> and a <a>Consumer</a> 
+                sharing a single <a>WoT Runtime</a>.
+            </p>
+        </section>
+        <section id="sec-topologies-deployment-scenarios">
+            <h2>WoT System's Topologies and Deployment Scenarios</h2>
```
Comment (mlagally) [inline index.html]: Topology of WoT systems ...?

File: index.html (hunk 1 above)
Comment (mlagally) [reply index.html]: A Consumer can be implemented on an edge device, in a gateway or in the cloud.

**Comment by mlagally** [comment]
approved in call on May 2nd.
rebase to candidate branch.

File: index.html (hunk 1 above)
Comment (takuki) [reply index.html]: Thank you for incorporating those changes on CR-candidate-3 branch!

**Comment by takuki** [comment]
Thank you for also making changes "NAT gateway" to "NAT device". I saw the changes are already in place.

**Pull request merged**